src/Module/Configuration/Form/CarriersForm.php

```diff
@@ -1262,6 +1262,41 @@ private function getExtraTabFields(Carrier $carrier, string $prefix = ''): array
                 'name'  => $prefix . Constant::INSURANCE_CONFIGURATION_NAME,
                 'desc'  => $this->module->l('Package will be insured according to below settings when Always insure package is on, or any product in the order has insurance set to on.', 'carriers'),
             ];
+            try {
+                $c = ConsignmentFactory::createByCarrierId($myParcelCarrier->getId());
```

Comment:
Wat is een `c`? 😛

Reply:
Argh, die had ik later beter genoemd, maar kennelijk niet in deze commit :-(